scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -311,11 +317,36 @@ class FossId(
                     }
                 }
 
-                checkScan(scanCode)
-                val rawResults = getRawResults(scanCode)
-                val resultsSummary = createResultSummary(startTime, rawResults)
+                val provenance = RepositoryProvenance(pkg.vcsProcessed, pkg.vcsProcessed.revision)
```

Comment:
Thanks, approved. Yet one more question, though: Do we currently support scanning source artifacts with FOSSID? Because if so, we can't hard-code the provenance like this here. And if not, we should support scanning source artifacts with FOSSID.

Reply:
Looking at the API documentation, the way to scan source artefacts is through the file upload mechanism we don't support (yet).
My suggestion would be to leave the provenance implementation as it is in current PR and tweak it when we implement file upload mode.
I created a ticket for it:
https://github.com/oss-review-toolkit/ort/issues/4242